src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -410,6 +438,7 @@ public boolean updateRepairSegment(RepairSegment newRepairSegment) {
     if(segmentRow != null){
       segment = createRepairSegmentFromRow(segmentRow);
     }
+    LOG.info("Refreshed segment {}", segment.getId());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "segment" is nullable here [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2259)